cmd/podman/root_cgroups_unsupported.go

```diff
@@ -0,0 +1,7 @@
+//go:build windows || darwin || freebsd
```

Comment:
(I suppose this could be `!linux` to cover all build possibilities exhaustively.)

Reply:
done